### PR TITLE
MODINVSTOR-435: Add indexes for callNumber and accessionNumber.

### PR DESCRIPTION
**Pull request by julianladisch**
Accents are not removed because they are relevant for libraries that use them.

### CODE REVIEW

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -582,6 +582,14 @@
           "fieldName": "hrid",
           "tOps": "ADD"
         }
+      ],
```
Comment (bohdan-suprun) [inline src/main/resources/templates/db_scripts/schema.json]: Do we need to update `fromModuleVersion` for the table then?

Comment (julianladisch) [reply src/main/resources/templates/db_scripts/schema.json]: No, indexes are created and updated regardless of `fromModuleVersion`.

Comment (bohdan-suprun) [reply src/main/resources/templates/db_scripts/schema.json]: Ok, what is the purpose of `fromModuleVersion` for table then? Is it required for table?

Comment (julianladisch) [reply src/main/resources/templates/db_scripts/schema.json]: Yes, when the value of "withMetadata" or "withAuditing" changes `fromModuleVersion` must be updated to the current version to trigger the change.

**Pull request merged**